stardog/admin.py

```diff
@@ -541,20 +541,15 @@ def new_virtual_graph(self, name, mappings, options={}, datasource=None, db=None
             with mappings.data() as data:
                 mappings = data.read().decode() if hasattr(data, 'read') else data
 
+        meta = {}
+        meta['name'] = name
+        meta['mappings'] = mappings
         if options:
-            meta = {
-                'name': name,
-                'mappings': mappings,
-                'options': options,
-            }
-        else:
-            meta = {
-                'name': name,
-                'mappings': mappings,
-                'data_source': datasource,
-                'db': db
-            }
-
+            meta['options'] = options
```

Comment:
I don't see any harm by changing `{}` to `None`, as an empty dict is falsy, hence `if options`, when no options is passed, will not add the options param. Since options field is optional, I guess it makes more sense to set all default values to None, so I updated this.
for the other part, iiuc adding "is not None" is redundant, and it's not adding anything in reality. maybe I am missing something?